File: 9-PalindromeNumber/9-PalindromeNumber.py
# Last updated: 11/7/2025, 12:00:09 PM
class Solution:
    def isPalindrome(self, x: int) -> bool:
        myString=str(x)
        for i in range(len(myString)//2):
            if myString[i]==myString[~i]:
                continue
            else: return False
        return True

File: 9-PalindromeNumber/9-PalindromeNumber.cpp
// Last updated: 11/7/2025, 12:00:08 PM
class Solution {
public:
    bool isPalindrome(int x) {
       if(x<0) return false;
        long long int ans = 0;
        int n = x, isDigit;
        while(n!=0){
            isDigit = n%10;
            ans = ans*10 + isDigit;
            n = n/10;
            }
        if(ans==x) return true;
        else return false;
        
    }
};